tests/garage/tf/samplers/test_off_policy_vectorized_sampler.py

```diff
@@ -63,9 +63,11 @@ def test_no_reset(self):
 
             assert len1 == 5 and len2 == 5, 'Sampler should respect batch_size'
 
-            assert len(paths1[0]['rewards']) + len(paths2[0]['rewards']) == \
-                paths2[0]['running_length'], (
+            # yapf: disable
+            assert (len(paths1[0]['rewards']) + len(paths2[0]['rewards'])
+                    == paths2[0]['running_length']), (
```

Comment:
yapf splits this as 

```
(len(paths1[0]['rewards']) + len(
paths2[0]['rewards']) == paths2[0]['running_length'])
```

Reply:
yapf: disable is a fine solution to this.